hrjobcontract/js/src/job-contract/controllers/modal/modal-contract-new.controller.js

```diff
@@ -254,6 +264,19 @@ define([
       }));
     }
 
+    /**
+     * Fetches list of providers
+     *
+     *  @return {Promise}
+     */
+    function fetchProviders () {
+      var providerGroups = [
```

Comment:
now that you have a "constant", i think you don't need this function (`fetchProviders`) at all anymore, you could directly do
```js
$q.all([
  fetchInsurancePlanTypes(),
  loadProviderOptions(Object.keys(PROVIDERS))
])
```